lib/turkish.dart

```diff
@@ -79,6 +83,10 @@ class _Turkish {
         : input.toLowerCase();
   }
 
+  int compareTr(String a, String b, bool ignoreCase) {
```

Comment:
Change the name of this method to `compareToTr` and private method `_compareTr` to `_compareToTr`. Private method name was not correct before.